src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/ContributorTeamRepository.cs

```diff
@@ -65,17 +66,19 @@ public List<ContributorTeam> Find()
         /// <param name="id">The id of contributor team to remove.</param>
         public void Remove(int id)
         {
-            throw new NotImplementedException();
+            var contributorTeamDelete = _unitOfWork.Context.ContributorTeams.Single(x => x.Id == id);
```

Comment:
What is the reason for this change?

Reply:
implementing this method instead of throwing exception. Revert changes?

Reply:
Yes, we do not need this functionality

Reply:
ok